pkg/controllers/v1alpha1/databackup/implement.go

```diff
@@ -304,6 +304,14 @@ func (r *DataBackupReconcilerImplement) generateDataBackupValueFile(ctx reconcil
 	nodeName, ip, rpcPort := utils.GetAddressOfMaster(masterPod)
 
 	imageName, imageTag := docker.GetWorkerImage(r.Client, databackup.Spec.Dataset, "alluxio", databackup.Namespace)
+	databackupImage := common.DEFAULT_ALLUXIO_DATA_BACKUPER_IMAGE
+	databackupImageInfo := strings.Split(databackupImage, ":")
+	if len(imageName) == 0 {
+		imageName = databackupImageInfo[0]
+	}
+	if len(imageTag) == 0 {
+		imageTag = databackupImageInfo[1]
+	}
```

Comment:
how about to use value from "GetImageRepoTagFromEnv" to overwrite default?

Reply:
Fixed it. Thanks